src/reducers/billing-reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Reducer function to handle billing
+ * @param {Object} billingState State values of billing
+ * @param {*} billingAction The changed state
+ * @returns Updated state into billingState
+ */
+const billingReducer = (billingState, billingAction) => {
```

Comment:
you can destructure this billingAction and use it instead of using dot operator everytime